joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/testfixtures/CSharpCode2CpgFixture.scala

```diff
@@ -26,10 +26,11 @@ class CSharpCode2CpgFixture(withPostProcessing: Boolean = false, withDataFlow: B
   protected def basicBoilerplate(
     contents: String = "Console.WriteLine(\"Hello, world!\");",
     namespace: String = "HelloWorld",
-    className: String = "Program"
+    className: String = "Program",
+    globalDeclarations: String = ""
   ): String =
     s"""using System;
-       |
+       |$globalDeclarations
```

Comment:
It seems like one can't define variables that don't belong to any class https://stackoverflow.com/a/14368428
Has this changed in recent .NET versions?